File: uri-online-judge/1008.go
package main

import "fmt"

var number int
var value float32
var hours float32

func main() {
	fmt.Scanf("%d\n%f\n%f", &number, &value, &hours)
	fmt.Printf("NUMBER = %d\n", number)
	fmt.Printf("SALARY = U$ %.2f\n", value*hours)
}

File: c++/1008.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
  int id, hours;
  double hourValue;

  cin >> id >> hours >> hourValue;
  cout << "NUMBER = " << id << endl;
  cout << setprecision(2) << fixed << "SALARY = U$ " << (hours * hourValue) << endl;
  return 0;
}